pubnub-core/src/data/presence.rs

```diff
@@ -0,0 +1,82 @@
+//! Presence API related types.
+use super::object::Object;
+use super::uuid::UUID;
+use std::collections::HashMap;
+
+pub mod respond_with {
+    //! Type system level flags to specialize the response types.
+
+    /// Return occupancy only.
+    #[derive(Debug, Clone, Copy)]
+    pub struct OccupancyOnly;
+    /// Return occupancy and UUIDs of the users.
+    #[derive(Debug, Clone, Copy)]
+    pub struct OccupancyAndUUIDs;
+    /// Return cooupance, UUIDs of the users and the related states.
```

Comment:
nitpick, typo:

```suggestion
    /// Return occupancy, UUIDs of the users and the related states.
```

Reply:
Thanks! I noticed this before, but couldn't find it. :+1: